README.md

```diff
@@ -1,3 +1,7 @@
+**We're working on the [next generation of Storage CPP SDK](https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage). It is currently in public preview and we expect to make it generally available soon. We encourage you to start evaluating the new SDK . The package can be found [here](https://github.com/Azure/azure-sdk-for-cpp/releases)**
```

Comment:
```suggestion
**We're working on the [next generation of Storage CPP SDK](https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage). It is currently in public preview and we expect to make it generally available soon. We encourage you to start evaluating the new SDK. The package can be found [here](https://github.com/Azure/azure-sdk-for-cpp/releases)**
```

We should consider linking to the getting started guide for the new SDK (for example point to the vcpkg acquisition) rather than just the list of releases.
Maybe here: https://github.com/Azure/azure-sdk-for-cpp#download--install-the-sdk